packages/react-microfrontend/src/helpers.tsx

```diff
@@ -0,0 +1,38 @@
+// const isVersionsDifferent = (versions1, versions2) => {
```

Comment:
I have commented on these helpers functions because it was making no sense, there were variables that didn't exist being used, I'd like to discuss if there is any need